README.md

```diff
@@ -115,7 +115,7 @@ If you are using L5Kit or dataset in your work please cite the following [whitep
 title = {One Thousand and One Hours: Self-driving Motion Prediction Dataset},
 author = {Houston, J. and Zuidhof, G. and Bergamini, L. and Ye, Y. and Jain, A. and Omari, S. and Iglovikov, V. and Ondruska, P.},
 year = {2020},
-howpublished = {\url{https://level5.lyft.com/dataset/}}
+howpublished = {\url{https://self-driving.lyft.com/}}
```

Comment:
this doesn't get replaced (it appears as `\url{https://self-driving.lyft.com/}`). Is it intended?

Reply:
yes